apps/backend/server.js

```diff
@@ -12,24 +14,33 @@ const downloadCourse = require('./src/routes/downloadCourse');
 const katex = require('./src/routes/katex');
 const sendEmail = require('./src/routes/sendEmail');
 
+sentry.init({
+  dsn: process.env.DSN || '',
```

Comment:
What is the `DNS`?

Reply:
it's like an api key to connect the app to my sentry account, you'd have to create an account for classroomio to get yours. (Data Source Name)